src/lib/mongoBackend/safeMongo.cpp

```diff
@@ -290,16 +290,17 @@ void setStringVector
       }
     }
   }
-
-  // Detect error
-  if (!b.hasField(field))
-  {
-    LM_E(("Runtime Error (object field '%s' is missing in BSONObj <%s> from caller %s:%d)", field.c_str(), b.toString().c_str(), caller.c_str(), line));
-  }
-  else
-  {
-    LM_E(("Runtime Error (field '%s' was supposed to be an array but type=%d in BSONObj <%s> from caller %s:%d)",
-          field.c_str(), b.getField(field).type(), b.toString().c_str(), caller.c_str(), line));
+  else {
```

Comment:
I think this is the casue of the errors recently reported by @iariasleon and @jmcanterafonseca